languages/elixir/concepts/atoms/about.md

```diff
@@ -1,7 +1,7 @@
-You can use [atoms][atom] whenever you have a set of constants you want to express. Using [atoms][atom] opens a type-safe way of interacting with constant values. An atom is defined by its name, following the [atom][atom] format:
```

Comment:
Agreed that this is a bit awkward on a re-read, but I think somewhere in our description of atoms we should indicate that they are symbols or something that represents a type-safe name.